README.md

```diff
@@ -1 +1,30 @@
-# e2e-latency-bench
\ No newline at end of file
+# E2E Latency Comparison Benchmark
+This benchmark measures E2E Latency across several blockchains. The numbers produced by the benchmark represent an end user’s observed latency from submitting a transaction to getting a transaction confirmation back. We compared the latency of coin transfer transactions across multiple blockchain networks in their mainnet environments. Coin transfer is a straightforward and cost-effective transaction type that is widely supported by SDKs.
+
+The benchmark leverages TS SDKs for the respective blockchains. The logic of 'Coin transfer' transaction for a blockchain is present in the corresponding 'index.js' file. Some common helper functions are present in 'common.js'.
+
+## Prerequisites
```

Comment:
probably also write that they will need a timeseries database that supports the prometheus remote_write protocol: https://prometheus.io/docs/concepts/remote_write_spec/, as that's what we're using to push metrics from each of the scripts.

Reply:
done